pkg/runtime/application_config_provider.go

```diff
@@ -19,7 +19,12 @@ const domains = "domains"
 const externalEvents = "externalEvents"
 
 var databaseConfig = config.MustRegisterSection(database, &interfaces.DbConfigSection{})
-var flyteAdminConfig = config.MustRegisterSection(flyteAdmin, &interfaces.ApplicationConfig{})
+
+var defaultFlyteAdminConfig = &interfaces.ApplicationConfig{
```

Comment:
this might be a time to create default configs for all the configs? Or if you want do it as a follow up PR?

Reply:
considering how many configs we have... how about a follow-up pr :)

Reply:
https://github.com/flyteorg/flyte/issues/904